manifests/s/Shodo-1/2/0/1.2.0/Shodo-1.2.0.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Shodo-1.2.0
+PackageVersion: 1.2.0
+PackageLocale: en-US
+Publisher: Assinador Shodo
+PackageName: Shodo-1.2.0

```

```suggestion
Publisher: Pje
PublisherUrl: https://ww2.trt2.jus.br/servicos/acesso-online/processo-judicial-eletronico-pje/page
PackageName: PJe.Sjodo
PackageUrl: https://www2.trt8.jus.br/PJe/Shodo%20-%20Processo%20Judicial%20Eletr%C3%B4nico.htm
```
"Assinador" just means "signer".